composer.json

```diff
@@ -14,19 +14,22 @@
         "php": ">=5.4",
         "psr/http-message": "^1.0",
         "php-http/message-factory": "^1.0.2",
-        "clue/stream-filter": "^1.3"
+        "clue/stream-filter": "^1.3",
+        "slim/slim": "^3.5"
```

Comment:
Should be yes. But this way it wont still work because. You cannot do the `composer install` and then `composer remove slim/slim` with PHP 5.4 anyway since it is the  `composer install` which  will fail. I try if `--ignore-platform-reqs` helps.